src/commands/site-list.php

```diff
@@ -167,32 +212,158 @@ function( $site ) {
 		}
 	}
 
-	protected function count_sites( $site_list, $term ) {
+	protected function eval_which_host( $site, $pressable_sites ) {
+		if ( true === $site->is_wpcom_atomic ) {
+			$server = 'Atomic';
+		} elseif ( true === $site->jetpack ) {
+			if ( in_array( parse_url( $site->URL, PHP_URL_HOST ), $pressable_sites ) ) {
+				$server = 'Pressable';
+			} else {
+				$server = 'Other';
+			}
+		} else {
+			$server = 'Simple'; // Need a better way to determine if site is simple. eg. 410'd Jurrasic Ninja sites will show as Simple.
+		}
+		return $server;
+	}
+
+	protected function filter_public_sites( $site_list ) {
+		$filtered_site_list = array();
+		foreach ( $site_list as $site ) {
+			if ( '' === $site[9] && '' === $site[4] && '' === $site[5] && ( 'is_subsite' !== $site[7] || '' !== $site[3] ) ) {
+				if ( '' === $site[2] || ( '' !== $site[2] && '' !== $site[3] ) ) {
+					$filtered_site_list[] = array(
+						$site[0],
+						$site[1],
+						$site[8],
+						$site[6],
+					);
+				}
+			}
+		}
+		return $filtered_site_list;
+	}
+
+	protected function eval_site_list( $site_list, $audit_type ) {
+		$audit_site_list = array();
+		foreach ( $site_list as $site ) {
+			if ( 'no-staging' === $audit_type && false !== strpos( $site[1], 'staging' ) ) {
+				continue;
+			}
+			if ( 'full' !== $audit_type && 'no-staging' !== $audit_type && ! in_array( $audit_type, $site, true ) ) {
+				continue;
+			}
+			if ( '' === $site[9] && '' === $site[4] && '' === $site[5] && ( 'is_subsite' !== $site[7] || '' !== $site[3] ) ) {
+				if ( '' === $site[2] || ( '' !== $site[2] && '' !== $site[3] ) ) {
+					$result = 'PASS';
+				} else {
+					$result = 'FAIL';
+				}
+			} else {
+				$result = 'FAIL';
+			}
+			$audit_site_list[] = array(
+				$site[0],
+				$site[1],
+				$site[2],
+				$site[3],
+				$site[4],
+				$site[5],
+				$site[7],
+				$site[9],
+				$site[6],
+				$result,
+				$site[8],
+			);
+		}
+		return $audit_site_list;
+	}
+
+	protected function eval_ignore_list( $site, $ignore ) {
+		$filtered_on = '';
+		foreach ( $ignore as $word ) {
+			if ( false !== strpos( $site->URL, $word ) ) {
+				$filtered_on = $word;
+				break;
```

Comment:
This is probably unnecessary, but I see that this function only returns the first match of the ignore list. 
I see that in some cases, there are sites that include more than one of these words and I'm wondering if it would be somehow useful to list all the matching words instead of the first one only.

For example `http://annaka-harris-development.mystagingwebsite.com/` is listed only as `staging` when it also includes the `development` word.

I understand that showing only `staging` is enough in cases like this, but I'm sharing this case here in case we consider that adding additional info to that column could benefit our audits.

Reply:
Great suggestion, added in https://github.com/Automattic/team51-cli/pull/159/commits/2c3314f4c921b8c485a5332a00d179c093b58a6c